Package.resolved

```diff
@@ -0,0 +1,16 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "CwlCatchException",
+        "repositoryURL": "https://github.com/mattgallagher/CwlCatchException.git",
+        "state": {
+          "branch": null,
+          "revision": "f809deb30dc5c9d9b78c872e553261a61177721a",
+          "version": "2.0.0-beta.1"
```

Comment:
Any reason we're targeting a beta instead of stable version?

Reply:
Version 2 is what adds support for SPM, there is no stable version of 2 yet